test/conformance/context/context_adapter_native_cpu.match

```diff
@@ -1,3 +1,3 @@
-urContextCreateWithNativeHandleTest.InvalidNullPointerDevices/SYCL_NATIVE_CPU___SYCL_Native_CPU_
-urContextCreateWithNativeHandleTest.InvalidNullPointerContext/SYCL_NATIVE_CPU___SYCL_Native_CPU_
-urContextSetExtendedDeleterTest.Success/SYCL_NATIVE_CPU___SYCL_Native_CPU_
+urContextCreateWithNativeHandleTest.InvalidNullPointerDevices/SYCL_NATIVE_CPU___SYCL_Native_CPU_____
```

Comment:
Should we use `{{.*}}` when matching the device name for native CPU? Just in case it changes or a UUID is added in future.